modules/page-memory/src/test/java/org/apache/ignite/internal/pagememory/persistence/PartitionMetaTest.java

```diff
@@ -41,15 +44,45 @@ void testLastAppliedIndex() {
 
         assertEquals(0, meta.lastAppliedIndex());
 
-        assertDoesNotThrow(() -> meta.lastAppliedIndex(null, 100));
+        assertDoesNotThrow(() -> meta.lastApplied(null, 100, 10));
 
         assertEquals(100, meta.lastAppliedIndex());
 
-        assertDoesNotThrow(() -> meta.lastAppliedIndex(UUID.randomUUID(), 500));
+        assertDoesNotThrow(() -> meta.lastApplied(UUID.randomUUID(), 500, 50));
 
         assertEquals(500, meta.lastAppliedIndex());
     }
 
+    @Test
+    void testLastAppliedTerm() {
+        PartitionMeta meta = new PartitionMeta();
+
+        assertEquals(0, meta.lastAppliedTerm());
+
+        assertDoesNotThrow(() -> meta.lastApplied(null, 100, 10));
```

Comment:
When you just call a method, it's not immediately clear what you want to check here. After a second of thinking you understand: oh, they are making sure it does not explode. On the other hand, using `assertDoesNotThrow()` makes this expectation explicit. I would prefer to leave it here.

Reply:
But this is not true: you actually check that `lastApplied` sets the index and the term, which is checked below using `assertEquals`

Reply:
Oops, I've confused the situation. Now I know the answer: I did like it's done because there is another test that was written before me about `index`. I wanted to avoid, ahem, stylistic differences.

Reply:
Can we change the other tests instead? This looks quite strange to me

Reply:
Removed it in first 3 test methods